v03_pipeline/lib/paths.py

```diff
@@ -338,22 +338,22 @@ def valid_reference_dataset_path(
             reference_dataset.access_control,
             reference_genome,
         ),
-        'reference_datasets',
```

Comment:
removed this constant since it's no longer necessary to disambiguate between query types.